src/handler.js

```diff
@@ -39,6 +39,10 @@ function resultFromRun(run) {
         return 'canceled';
       }
 
+      if (run.reasonResolved === 'superseded') {
+        return 'coalesced';
```

Comment:
Hrm, I could have sworn I submitted a review to you on this already.  Sheesh, so sorry.  We have actually decided to migrate to your term ``superseded`` internally in Treeherder.  I have merged my PR that uses (and expects) that term as a result.  So if you'd switch this to ``superseded`` here and in your version of the ``yml`` then we'll be good to go.  

My PR is on stage now, and should be on prod Tuesday next week, I expect.